fineract-provider/src/test/java/org/apache/fineract/mix/report/service/XBRLBuilderTest.java

```diff
@@ -79,10 +79,13 @@ public void shouldCorrectlyBuildMap() {
             nodes = DocumentBuilderFactory.newInstance().newDocumentBuilder().parse(new ByteArrayInputStream(result.getBytes()))
                     .getElementsByTagName("Assets");
         } catch (final SAXException e) {
+            // TODO Auto-generated catch block
```

Comment:
@xurror this looks weird and given (your) #663 I'm assuming is a mistake - remove?

```suggestion
```